page/partials/commit_funds.ejs

```diff
@@ -0,0 +1,51 @@
+<div class="commit-funds section">
+        <div class="row">
+            <div class="col-md-6">            
+                <div class="number">
+                    &nbsp;&nbsp;&nbsp;&nbsp;01
```

Comment:
wtf is that? xD I think using paddings to artificially expand element is a better idea.